packages/preview-middleware/src/base/flex.ts

```diff
@@ -90,6 +99,16 @@ export function writeChange(
         }
         const filePath = join(path, fileName + '.' + fileType);
         writeFileSync(filePath, JSON.stringify(data, null, 2));
+        const telemetryData = {
+            category: 'Save',
+            changeType: data.changeType,
+            controlType: data?.selector?.type ?? '',
+            propertyName: data?.content?.property ?? '',
```

Comment:
We probably need to add a test case to cover these branches to reach the required code coverage.

Reply:
covered